src/common/sync-box/channel/enlist-channel-listener-injection-token.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectionToken } from "@ogre-tools/injectable";
+
+export const enlistChannelListenerInjectionToken = getInjectionToken<
+  (channel: any, handler: any) => () => void
```

Comment:
And again here

Reply:
Typings for these came in latter commit. Sorry for not rebasing.